clustered/clustered-dist/src/assemble/server/conf/tc-config.xml

```diff
@@ -1,61 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<tc-config xmlns="http://www.terracotta.org/config"
```

Comment:
The fact that there is no "default" configuration means that OSS users can no longer just run `./start-tc-server.sh` (or it's replacement) and have a functional 'cluster' of one machine. Do we want to consider having a default properties file equivalent?

Reply:
This is explained in the presentation, slide 27.
Initially, the goal was to have another script, to highlight this UX change.
Now, keeping the same script, we will have a lot of fuzzy questions like that, part of teh reason I don't like the idea to re-use old stuff.

The answer IMO would be: NO.

Now, the way a server starts, it starts unconfigured. User has to configure it, or start it pre-activated with a cluster name or a config file, and this needs to be documented. **The behavior to start & configure a server has changed.**

We cannot _by default_ have it run `./start-tc-server.s` and make the user get a 1-node activated stripe because he won't be able to use the same script to start an un-configured node and build his cluster (which should be the default, as it was decided 1 year ago with everybody). And it would nott be the same behavior as EE also.

So what we could do though, is provide a default `cluster.properties` in the `conf` fodler that is showing all configurable settings. We could add a default cluster-name there and if the user is pointing to this file att startup, it would pre-activate the node.

But anyway, he will have to know about the `--node-repository-dir` setting, which is not part of the configuration file, and this is one of the most important setting to know now.

Reply:
Added a default config file:

```properties
#Configuration for node 'covid' in stripe ID 1
client-lease-duration=150s
client-reconnect-window=120s
cluster-name=cluster
failover-priority=availability
offheap-resources=primary-server-resource:512MB
stripe.1.node.1.node-bind-address=0.0.0.0
stripe.1.node.1.node-group-bind-address=0.0.0.0
stripe.1.node.1.node-group-port=9430
stripe.1.node.1.node-hostname=localhost
stripe.1.node.1.node-log-dir=%H/terracotta/logs
stripe.1.node.1.node-logger-overrides=
stripe.1.node.1.node-name=covid
stripe.1.node.1.node-port=9410
stripe.1.node.1.node-public-hostname=
stripe.1.node.1.node-public-port=
stripe.1.node.1.tc-properties=
```

Usage:

```
~/.../distributions/ehcache-clustered-3.9-SNAPSHOT-kit ( kit → origin ↑1 ↓1 {8} U:1 ⊘ )
❯  ./server/bin/start-tc-server.sh --config-file server/conf/cluster.properties 
2020-04-07 18:42:34,040 INFO - PID is 25189
2020-04-07 18:42:35,049 INFO - Available Max Runtime Memory: 1822MB
2020-04-07 18:42:35,660 INFO - Terracotta Server instance has started up as ACTIVE node on 0:0:0:0:0:0:0:0:9410 successfully, and is now ready for work.
2020-04-07 18:42:35,660 INFO - Moved to State[ ACTIVE-COORDINATOR ]

```